app/main/helpers/frameworks.py

```diff
@@ -65,7 +64,7 @@ def get_required_fields(all_fields, answers):
         required_fields.add('SQ3-1k')
 
     # If you answered No to question 26 (established in the UK)
-    if not answers.get('SQ5-2a'):
+    if answers.get('SQ5-2a') is False:
```

Comment:
What about `if 'SQ5-2a' in answers and not answers['SQ5-2a']`. It's more verbose, but I think it describes what we're checking better and it avoids the identity check